src/dcd_mapping/annotate.py

```diff
@@ -46,29 +46,17 @@
 _logger = logging.getLogger(__name__)
 
 
-def _allele_to_v1_allele(allele: Allele) -> vrs_v1_schemas.Allele:
-    """Convert VRS 2.0 allele to VRS 1.3 allele.
-
-    :param allele: VRS 2.0a allele
-    :return: equivalent VRS 1.3 allele
-    """
-    start = allele.location.start
-    end = allele.location.end
-    sequence_id = f"ga4gh:{allele.location.sequenceReference.refgetAccession}"  # type: ignore
-    location_raw = f'{{"end":{{"type":"Number","value":{end}}},"sequence_id":"{sequence_id.split(".")[1]}","start":{{"type":"Number","value":{start}}},"type":"SequenceLocation"}}'
-    location_id = sha512t24u(location_raw.encode("ascii"))
-    sequence = "" if not allele.state.sequence else allele.state.sequence.root
-    allele_raw = f'{{"location":"{location_id}","state":{{"sequence":"{sequence}","type":"LiteralSequenceExpression"}},"type":"Allele"}}'
-    allele_id = sha512t24u(allele_raw.encode("ascii"))
-
+def _allele_to_1_3_allele(allele: Allele) -> vrs_v1_schemas.Allele:
+    """Convert VRS 2.0 allele to 1.3 allele instance."""
+    sequence = allele.state.sequence.root if allele.state.sequence else ""
     return vrs_v1_schemas.Allele(
-        id=f"ga4gh:VA.{allele_id}",
+        id=ga4gh_identify(allele, as_version=PrevVrsVersion.V1_3),
         location=vrs_v1_schemas.SequenceLocation(
-            id=location_id,
-            sequence_id=sequence_id,
+            id=ga4gh_identify(allele.location, as_version=PrevVrsVersion.V1_3),
```

Comment:
@jarbesfeld notes that we previously weren't providing VSL IDs in the provided objects. can/should we leave this out of the 1.3 objects?